concrete/tools/sitemap_search_selector.php

```diff
@@ -25,72 +28,72 @@
 */
 
 ?>
-<div class="ccm-ui" id="ccm-sitemap-search-selector">
 
-<?=Loader::helper('concrete/ui')->tabs(array(
-    array('sitemap', t('Full Sitemap')),
-    array('explore', t('Flat View')),
-    array('search', t('Search')),
-));
-?>
+<div class="ccm-ui" id="ccm-sitemap-search-selector">
 
-<div id="ccm-tab-content-sitemap" class="ccm-tab-content"></div>
+	<?php
+        echo $app->make('helper/concrete/ui')->tabs([
+            ['sitemap', t('Full Sitemap')],
+            ['explore', t('Flat View')],
+            ['search', t('Search')],
+        ]);
+    ?>
 
+	<div id="ccm-tab-content-sitemap" class="ccm-tab-content"></div>
 
-<div id="ccm-tab-content-explore" class="ccm-tab-content"></div>
+	<div id="ccm-tab-content-explore" class="ccm-tab-content"></div>
 
-<div id="ccm-tab-content-search" class="ccm-tab-content"></div>
+	<div id="ccm-tab-content-search" class="ccm-tab-content"></div>
 
 </div>
 
 <script type="text/javascript">
-ccm_sitemapSearchSelectorHideBottom = function() {
-	$('#ccm-sitemap-search-selector').parent().parent().find('.ui-dialog-buttonpane').hide();
-}
-
-ccm_sitemapSearchSelectorShowBottom = function() {
-	$('#ccm-sitemap-search-selector').parent().parent().find('.ui-dialog-buttonpane').show();
-}
-
+	ccm_sitemapSearchSelectorHideBottom = function() {
+		$('#ccm-sitemap-search-selector').parent().parent().find('.ui-dialog-buttonpane').hide();
+	}
 
-$(function() {
-	var sst = jQuery.cookie('ccm-sitemap-selector-tab');
-	if (sst != 'explore' && sst != 'search') {
-		sst = 'sitemap';
+	ccm_sitemapSearchSelectorShowBottom = function() {
+		$('#ccm-sitemap-search-selector').parent().parent().find('.ui-dialog-buttonpane').show();
 	}
-	$('a[data-tab=' + sst + ']').parent().addClass('active');
-	ccm_sitemapSearchSelectorHideBottom();
-	$('a[data-tab=sitemap]').click(function() {
-		jQuery.cookie('ccm-sitemap-selector-tab', 'sitemap', {path: '<?=DIR_REL?>/'});
-		ccm_sitemapSearchSelectorHideBottom();
-		if ($('#ccm-tab-content-sitemap').html() == '') {
-			jQuery.fn.dialog.showLoader();
-			$('#ccm-tab-content-sitemap').load('<?=REL_DIR_FILES_TOOLS_REQUIRED?>/sitemap_overlay', function() {
-				jQuery.fn.dialog.hideLoader();
-			});
+
+	$(function() {
+		var sst = jQuery.cookie('ccm-sitemap-selector-tab');
+		if (sst != 'explore' && sst != 'search') {
```

Comment:
maybe we can 'improve' this by using strict comparison?

Reply:
No problem, but just to understand the logic, could you explain why a strict comparison is better here ?

Reply:
For one, because it looks better in your IDE ;)
![afbeelding](https://user-images.githubusercontent.com/1431100/46851171-e3112280-cdf6-11e8-8a25-d3a5d5450ad8.png)

Personally, when I see `!=` instead of `!==` then I assume that the one who wrote the code thinks that the types between both sides _could_ differ. But if that's not the case, we should just use strict comparison, imo.

Reply:
To make the IDE happy is a good enough reason 😄
Thank you for your explanation !